examples/platform/linux/CommissioneeShellCommands.cpp

```diff
@@ -84,6 +88,21 @@ static CHIP_ERROR CommissioneeHandler(int argc, char ** argv)
         return error  = SendUDC(true, chip::Transport::PeerAddress::UDP(commissioner, port));
     }
 #endif // CHIP_DEVICE_CONFIG_ENABLE_COMMISSIONER_DISCOVERY_CLIENT
+    else if (strcmp(argv[0], "restartmdns") == 0)
+    {
+        bool commissioningMode = false;
+        bool additionalPairing = false;
+        if (strcmp(argv[1], "true") == 0)
```

Comment:
is argc guaranteed to always be at least 3?

Wondering if this is easier to read:

```cpp
bool commisioningMode      =  (argc > 1) && (strcmp(argv[1], "true") == 0);
bool AdditionalComisioning =  (argc > 2) && (strcmp(argv[2], "true") == 0);
```

Reply:
Just realized that this is not the same as additional stays false if commisioning mode is false.

I believe this makes a stronger case of bitfields and the argument here should be just one mode, because the pair "true/false" is invalid.

Reply:
changed to enum from bitfield